real_api_test.py

```
@@ -0,0 +1,221 @@
+    with open(env_path) as f:
+        for line in f:
+            line = line.strip()
+            if line and not line.startswith('#') and '=' in line:
+                key, value = line.split('=', 1)
+                os.environ[key] = value
+
+# Add src to path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'src'))
+
+from mad_spark_alt.core.enhanced_orchestrator import EnhancedQADIOrchestrator
+from mad_spark_alt.core import SmartQADIOrchestrator
```

Ruff (0.11.9)

Module level import not at top of file

(E402)